lib/db/entity/sub_category.dart

```diff
@@ -0,0 +1,40 @@
+class SubCategory{
+  static const table = 'subcategory';
+  static const category_idCol = 'category_id';
+  static const sub_category_idCol = 'sub_category_id';
+  static const orderCol = ' order';
+
+  String category_id;
```

Comment:
Please follow Dart naming conventions for variables. They should be in camelCase